### DISCUSSION TITLE
Website Feature: Ability to hide old usernames.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20749] [Polls]
I think adding the option to hide your old usernames like you online status would be cool. What about you?

**Comment by Doksha** [comment]
if you really have a reason to remove an old name (ie personal information or maybe something inapropriate), you should be able to contact support to have it removed, otherwise, hiding old names could cause issues with identifying players. 
It would be cool, but I dont think something like this would be added.

**Reply by ghost** [reply]
Makes sense.

**Reply by 12beesinatrenchcoat** [reply]
They already do this (see https://osu.ppy.sh/wiki/en/Help_centre/Account#hide-username-history)
> If one of your previous usernames breaches your privacy (for example, if it contains your legal name) or can lead to your privacy being breached (via other websites), contact accounts@ppy.sh to request it removed.

**Reply by ghost** [reply]
No it has to breach your privacy for support to remove old username history

**Reply by 12beesinatrenchcoat** [reply]
I'm… confused. I was replying to the comment. A breach of privacy includes a legal name, personal information, or something of the sort, in which case support removes the username, as this comment states. Elaborate?

**Comment by peppy** [comment]
https://osu.ppy.sh/wiki/en/Help_centre/Account#hide-username-history
